GUI/GeneratorDialog.xaml.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.IO;
+using System.Windows;
+using BTSPEngine;
+
+namespace GUI;
+
+public partial class GeneratorDialog : Window
+{
+	public GeneratorDialog()
+	{
+		InitializeComponent();
+	}
+
+	private void GenerateButton_OnClick(object sender, RoutedEventArgs e)
+	{
+		var vertexCount = int.Parse(VertexCountInput.Text);
+		var maxEdgeWeight = double.Parse(MaxWeightInput.Text);
+		var generator = new BTSPGenerator(maxEdgeWeight);
+		var instanceCount = int.Parse(InstanceCountInput.Text);
+
+		const string dirName = "generated-examples";
+		Directory.CreateDirectory(dirName);
+
+		int i0 = 0;
+		while (File.Exists($"{MakeFileName(dirName, vertexCount, maxEdgeWeight, i0)}.graphml"))
+			i0++;
+
+		for (int i = 0; i < instanceCount; i++)
+		{
+			generator.GenerateToFile(vertexCount, MakeFileName(dirName, vertexCount, maxEdgeWeight, i0 + i));
```

Comment:
Jak każesz mu wygenerować 200 instancji po 500 wierzchołków to się zawiesza, można to owinąć w jakieś `Task.Run()`, dodać napis, że generowanie jest w trakcie i dodać `CancellationToken`, żeby można było to przerwać zamknięciem okna jak ktoś się rozmyśli